src/components/ImportForm/SampleSection/SampleSection.tsx

```diff
@@ -76,20 +76,20 @@ const SampleSection = ({ onStrategyChange }) => {
     (item) => {
       setSelected((prevState) => {
         if (prevState?.name === item.name) {
-          setFieldValue('source', undefined);
+          setFieldValue('source.git.url', undefined);
           return undefined;
         }
-        const sourceUrl = item?.attributes?.git?.remotes?.origin;
+        const newSourceUrl = item?.attributes?.git?.remotes?.origin;
 
-        setFieldValue('source', sourceUrl);
+        setFieldValue('source.git.url', newSourceUrl);
```

Comment:
Sample validation schema needs to be updated as well.

Reply:
Updared